src/Microsoft.AspNet.Mvc.Core/Controller.cs

```diff
@@ -1198,7 +1198,7 @@ public virtual Task<bool> TryUpdateModelAsync<TModel>([NotNull] TModel model)
         [NonAction]
         public virtual bool TryValidateModel([NotNull] object model)
         {
-            return TryValidateModel(model, prefix: null);
+            return TryValidateModel(model, prefix: string.Empty);
```

Comment:
Is this sufficient? I would think that the fix should really be in one place in ```ModelBindingHelper``` which treats null prefixes as empty value. With this a user can still pass null (since there is no NotNull attribute on the TryUpdate, TryValidate versions for back compat).
@dougbu do you remember why we chose to allow null values for prefixes?